### DISCUSSION TITLE
SDR++ macOS version for Apple Silicon

### DISCUSSION THREAD

**Discussion by betzburger** [AlexandreRouma/SDRPlusPlus#869] [General]
In my case this is what I did to compile it:

< removed > 

Please see the CI script for how to build the bundle

**Comment by AlexandreRouma** [comment]
Hi,

Some of these instructions are not advisable:
1) Never edit any file in the codebase. For cmake use `-DOPTION=VALUE` (eg. `-DUSE_BUNDLE_DEFAULTS=ON`)
2) Never `make install` SDR++ on MacOS. Especially when enabling bundle defaults. It is meant to be built into a bundle, NEVER installed. It is garanteed to break or cause issues when trying to uninstall.

After the bundle is built, it can be installed and run like any other MacOS bundle.

Please either edit this post to fix it or remove it before people try to follow it.
See the CI script for how to build the bundle, it's a single script to execute.
